test/test_http11.rb

```diff
@@ -183,4 +183,30 @@ def test_horrible_queries
     end
 
   end
+
+  def test_13632_workaround
```

Comment:
not sure I understand this situation. Is `purge_interrupt_queue` defined for any other ruby version? This test doesn't make sense if method undefined